sleap/gui/dialogs/formbuilder.py

```diff
@@ -224,6 +224,28 @@ def get_results(self) -> Optional[Dict[Text, Any]]:
         return self._results
 
 
+# We are writing this method to override the spinbox in add_item so the increment/decrement is x10/x0.1
+class ExponentialSpinBox(QtWidgets.QDoubleSpinBox):
+    def stepBy(self, steps: int):
+        currValue = self.value()
+
+        if steps == 0:
+            return
+        
+        if steps > 0:
+            newValue = currValue * 10
+            if currValue == 0:
+                newValue = 0.01
+
+        elif steps < 0:
+            newValue = currValue * ((10) ** -1)
```

Comment:
So, I believe interacting through the spinbox though it's different modes:
- buttons
- arrow keys
- mouse wheel

will pass in a different value for `steps`. I haven't tested this manually to see how if feels, but if we are multiplying/dividing by a factor of 10, then I like the current logic that only uses the +/- of `steps` to determine the output value. However, I think we should still manually test the different modes to see if we like it or not.